examples/time_series/MvGaussianRandomWalk_demo.ipynb

```diff
@@ -2,6 +2,7 @@
  "cells": [
```

Comment:
<p>note to self (or to however has time to check): I am not sure sphinx/myst support the <code>\begin{}</code> latex syntax out of the box, we may need some extra tweaks/configuration. I vaguely recall reading something about that, so I am quite sure it won't be a problem once configured and that how to configure that will be somewhere in https://myst-parser.readthedocs.io/en/latest/, maybe in optional syntaxes?</p><p> </p><p>myst should also support adding labels to equations IIRC which I think will be more convenient than the manual star sign.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
This has an example of a sphinx label used on one equation: https://myst-parser.readthedocs.io/en/latest/syntax/optional.html#dollar-delimited-math

And it is indeed possible to use `\begin` syntax directly (see https://myst-parser.readthedocs.io/en/latest/syntax/optional.html#syntax-amsmath), but I'm not sure if labels will work. I think for now the best would be to put the latex code inside a double dollar env or a math directive as shown in the first example